docs/_spec/08-pattern-matching.md

```diff
@@ -17,6 +17,8 @@ chapter: 8
                     |  Pattern3
   Pattern3        ::=  SimplePattern
                     |  SimplePattern {id [nl] SimplePattern}
+                    |  GivenPattern
+  GivenPattern    ::=  ‘given’ TypePat
```

Comment:
The parser considers that this a case of `SimplePattern`:
https://github.com/lampepfl/dotty/blob/7f410aa1b5cbb38dc4c51da0b5772c7babfaabf0/compiler/src/dotty/tools/dotc/parsing/Parsers.scala#L3026

Also, we should update `docs/_docs/internals/syntax.md` at the same time, to keep them in sync.